tasty/entities.py

```diff
@@ -0,0 +1,124 @@
+from copy import deepcopy
+from uuid import uuid4
+
+from rdflib import Namespace, RDF
+
+import tasty.graphs as tg
+import tasty.constants as tc
+
+
+class Instance:
```

Comment:
I think generally all of the classes in this file need some documentation.  Items like bind aren't necessarily clear out of context .

Reply:
Agreed. https://github.com/NREL/tasty/issues/7